pom.xml

```diff
@@ -481,8 +455,8 @@
       <artifactId>jersey-mvc</artifactId>
       <exclusions>
         <exclusion>
-          <groupId>javax.servlet</groupId>
-          <artifactId>servlet-api</artifactId>
+          <groupId>jakarta.servlet</groupId>
+          <artifactId>jakarta.servlet-api</artifactId>
```

Comment:
Need to properly exclude the new libraries, javax is no longer asked for and this servlet-api will conflict with the toolchain version from jetty